lookout/core/lib.py

```diff
@@ -9,21 +9,29 @@
 from lookout.core.api.service_data_pb2 import File
 
 
-def find_new_lines(before: File, after: File) -> List[int]:
+def find_modified_lines(before: File, after: File) -> List[int]:
     """
-    Return the new line numbers from the pair of "before" and "after" files.
+    Return the modified line numbers from the pair of "before" and "after" files.
 
+    If line was deleted neighbour lines considered as modified.
     :param before: the previous contents of the file.
     :param after: the new contents of the file.
-    :return: list of line numbers new to `after`.
+    :return: list of line numbers modified to `after`.
     """
-    matcher = SequenceMatcher(a=before.content.decode("utf-8", "replace").splitlines(),
-                              b=after.content.decode("utf-8", "replace").splitlines())
+    before_lines = before.content.decode("utf-8", "replace").splitlines()
+    after_lines = after.content.decode("utf-8", "replace").splitlines()
+    matcher = SequenceMatcher(a=before_lines, b=after_lines)
```

Comment:
before_lines/after_lines change is not needed and does not improve anything

Reply:
I use `after_lines` here: https://github.com/src-d/style-analyzer/pull/265/files/e0e436f5e6906d6f0fa82f2abae7ad61d7584fa5#diff-a7589e8177477e8366513b2d10ffeb6bR31